templates/show-worklist.html

```diff
@@ -14,6 +14,10 @@
     var awesomplete_by_username;
 
     $(document).ready(function(){
+
+    	// making the nav active
```

Comment:
This comment doesn't add much to what I can already read on the line below. Remove?

Reply:
Done.